homeassistant/components/overkiz/water_heater_entities/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from pyoverkiz.enums.ui import UIWidget
 
+from .atlantic_dhw import AtlanticDomesticHotWaterProductionMBLComponent
```

Comment:
Please rename the file to the full class name (as the other files). I am afraid that in the future we will get a 'AtlanticDHW' class as well.

Reply:
done